File: 448-find-all-numbers-disappeared-in-an-array/find-all-numbers-disappeared-in-an-array.py
class Solution(object):
    def findDisappearedNumbers(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        arr=[0]*(len(nums)+1)
        for x in nums:
            arr[x]+=1
        res=[]
        for i in range(1,len(nums)+1):
            if arr[i]==0:
                res.append(i)
        return res

File: 448-find-all-numbers-disappeared-in-an-array/find-all-numbers-disappeared-in-an-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var findDisappearedNumbers = function(nums) {
    let arr=new Array(nums.length+1).fill(0);
    let res=[];
    for(let x of nums){
        arr[x]++;
    }
    for(var i=1;i<=nums.length;i++){
        if(arr[i]==0){
            res.push(i);
        }
    }
    return res;
};